.github/workflows/close-as-support.yml

```diff
@@ -0,0 +1,50 @@
+name: Close as support issue
+permissions:
+  issues: write
+
+on:
+  workflow_call:
+    inputs:
+      number:
+        type: number
+        required: true
+      reason:
+        type: string
+        required: true
+
+jobs:
+  close-as-support:
+    runs-on: ubuntu-latest
+    env:
+      BODY: >
+        ${{ inputs.reason }}
```

Comment:
```suggestion
       A GitHub Desktop team member has looked at this issue and believes it is
        not the result of a bug in the application itself but rather an issue
        with your local environment. 
        
        ${{ inputs.reason }}
```

Reply:
(Unless we expect that a support issue may not be a local environment thing?)

Reply:
Maybe no newline after `More specifically`?